firmware/src/main.cpp

```diff
@@ -54,15 +54,15 @@ const mPMTTestStandConfig conf = {
     },
     .gantry = {
         .axis_mech = {
-            .counts_per_rev     = 500,    // For Igus gantry encoders
-            .steps_per_rev      = (200*4) // 200 * microstep = 200 * 4
+            .counts_per_rev     = ENCODER_COUNTS_PER_REV,
+            .steps_per_rev      = MOTOR_STEPS_PER_REV,
         },
-        .accel                  = 8000,   // acceleration for all motion [steps / s^2]
-        .vel_start              = 500,    // starting velocity for all motion [steps / s]
-        .vel_home_a             = 10000,  // holding velocity for homing A [steps / s]
-        .vel_home_b             = 10000,  // holding velocity for homing B [steps / s]
-        .accel_home_a           = 5000,   // acceleration for homing A [steps / s^2]
-        .accel_home_b           = 5000    // acceleration for homing B [steps / s^2]
+        .accel                  = 10,  // acceleration for all motion [steps / s^2]
+        .vel_start              = 1,   // starting velocity for all motion [steps / s]
+        .vel_home_a             = 75,  // holding velocity for homing A [steps / s]
+        .vel_home_b             = 75,  // holding velocity for homing B [steps / s]
+        .accel_home_a           = 10,  // acceleration for homing A [steps / s^2]
+        .accel_home_b           = 10   // acceleration for homing B [steps / s^2]
```

Comment:
How were these values chosen and why are they so much smaller than the previous `steps/s` values?

Reply:
They were chosen fairly arbitrarily. They're for the new gantry so there's no gear reduction and we need way lower speeds than before